modules/config.js

```diff
@@ -0,0 +1,20 @@
+const fromEntries = require('object.fromentries');
+
+const argsArray = process.argv.slice(2);
+const mapped = argsArray.map(argPair =>
+  argPair.split('=')
+    .map((item, index) => index ? item : item.slice(1))
+);
+const args = fromEntries(mapped);
+
+const envsObj = process.env;
+const appEnvsArr = Object.entries(envsObj).filter(item => item[0].startsWith('APP'));
+const appEnvs = fromEntries(appEnvsArr);
```

Comment:
I think this is an overkill to install vendor lib for this when you can do it with one line of code
`Object.entries(envsObj).reduce((res, [key, val]) => key.startsWith('APP') ? { ...res, [key]: val } : res, {});`

Reply:
agree

Reply:
just was surprised that Object.fromEntries() doesn't work in Node till v12